gensim/models/logentropy_model.py

```diff
@@ -46,9 +48,15 @@ class LogEntropyModel(interfaces.TransformationABC):
     """
 
     def __init__(self, corpus, normalize=True):
-        """
-        `normalize` dictates whether the resulting vectors will be
-        set to unit length.
+        """Constructs an instance of LogEntropyModel, initializes internal statistics of the corpus
+        and calls `initialize` method automatically.
+
+        Parameters
+        ----------
+        corpus : iterable object
+            itertable consisting of word-documents that are made up of term id and term frequency.
+        normalize : bool, optional
+            `normalize` dictates whether the resulting vectors will be set to unit length.
```

Comment:
`normalize` -> `parameter`, also it will be good if you present both cases

`if True, ..., otherwise ....`

Reply:
done